cips/discourage-memo-usage.md

```diff
@@ -0,0 +1,139 @@
+
+### `auth.TxSizeCostPerByte`
+
+`auth.TxSizeCostPerByte` is the gas cost per byte of a transaction. The current value of 10 is a Cosmos SDK default and it is comparable to the current `blob.GasPerBlobByte` value of 8. In order to discourage the usage of the memo field and encourage the use of blob data, we propose increasing `auth.TxSizeCostPerByte` to 16 so that each memo byte costs roughly twice as much as a blob byte.
+
+It is worth noting that `auth.TxSizeCostPerByte` is important outside the context of transactions memos because this parameter is used for all transaction bytes. Non-memo transaction contents may similarly bloat the `TRANSACTION_NAMESPACE`. For example, [ICS-020](https://github.com/cosmos/ibc/blob/0da326fbedfd2c96aad807ed25e6eafd1399db07/spec/app/ics-020-fungible-token-transfer/README.md?plain=1#L46) token transfers may contain an unbounded memo field. We can expect the usage and size of this memo field to increase if Packet Forward Middleware is adopted (see [CIP-9](./cip-9)).
+
+How expensive are transactions after a `auth.TxSizeCostPerByte` increase?
+
+`auth.TxSizeCostPerByte` | MsgSend without memo | MsgSend with 256 character memo | MsgPFB with 256 byte blob
+-------------------------|----------------------|---------------------------------|----------------------
+10                       | 77004 gas            | 79594 gas                       | 67765 gas
+16                       | 78906 gas            | 83050 gas                       | 69763 gas
+100                      | 105534 gas           | 131434 gas                      | 97735 gas
+1000                     | 390834 gas           | 649834 gas                      | 397435 gas
+
+Assuming `minimum-gas-prices = "0.002utia"`
+
+`auth.TxSizeCostPerByte` | MsgSend without memo | MsgSend with 256 character memo | MsgPFB with 256 byte blob
+-------------------------|----------------------|---------------------------------|----------------------
+10                       | 154 utia             | 159 utia                        | 135 utia
+16                       | 157 utia (+2%)       | 166 utia (+4%)                  | 139 utia (+3%)
+100                      | 211 utia (+37%)      | 262 utia (+65%)                 | 195 utia (+44%)
+1000                     | 781 utia (+407%)     | 1299 utia (+716%)               | 794 utia (+488%)
+
+Therefore, increasing from 10 to 16 is a conserative increase.
+
+### FAQ
+
+What do other blockchains use for these params?
+
+Param                    | Celestia | Cosmos Hub | Osmosis
+-------------------------|----------|------------|--------
+`auth.MaxMemoCharacters` | 256      | 512        | 256
+`auth.TxSizeCostPerByte` | 10       | 10         | 10
+
+## Backwards Compatibility
+
+This proposal is backwards compatible. However, clients that hard-coded gas estimation based on the previous `auth.TxSizeCostPerByte` value will need to be updated.
+
```

Comment:
- [ ] Add Test Cases section for testing this on Mocha testnet first

Reply:
Resolved via 68d1080844120f285b5fe6a3d78eb3e4c9e83054